safesocket/safesocket_darwin.go

```diff
@@ -83,6 +86,11 @@ func localTCPPortAndTokenDarwin() (port int, token string, err error) {
 				}
 			}
 		}
+		if os.Getuid() == 0 {
```

Comment:
I think you should use `os.Geteuid` here

Reply:
TIL